src/main/resources/application.properties

```
@@ -0,0 +1,21 @@
+server.port=8090
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.show_sql=true
+spring.jpa.properties.hibernate.format_sql=true
+spring.datasource.url=jdbc:mysql://localhost:3306/comexdb?USESSL=false
+spring.datasource.username=root
+spring.datasource.password=pablo123
+spring.jpa.database-platform=org.hibernate.dialect.MySQL5InnoDBDialect
```

Você está usando o MySQL 5? Estou usando o MySQL 8 por aqui, aí mudei para:

```suggestion
spring.jpa.database-platform=org.hibernate.dialect.MySQL8Dialect
```